diff --git a/configure.pri b/configure.pri
index 98c52d26..fc68119f 100644
--- a/configure.pri
+++ b/configure.pri
@@ -173,6 +173,10 @@ defineReplace(CheckClangLlvmLibForLupdateParser) {
                      LLVMRemarks
     }
 
+    versionIsAtLeast($$CLANG_VERSION, "10.0.0") {
+        libToTest += LLVMFrontendOpenMP
+    }
+
     CLANG_LLVM_LIBS =
     for (lib, libToTest) {
         libFullPath =
